examples/nodejs_wav/.gitignore

```diff
@@ -0,0 +1 @@
+models
```

Comment:
I'm not sure we want that

Reply:
Oh yeah, now that I think about it, it may be confusing if someone has a models directory.